.github/scripts/rust/Makefile.lint.toml

```diff
@@ -0,0 +1,8 @@
+[tasks.update-clippy]
+script_runner = "@rust"
+script = { file = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/lint.rs", absolute_path = true }
+
+[tasks.update-clippy-check]
+private = true
+env = { CARGO_MAKE_CLIPPY_LINT_MODE = "check" }
```

Comment:
```suggestion
env = { CARGO_MAKE_CLIPPY_LINT_MODE = "generate" }
```

Reply:
Why?

Reply:
We don't want clippy to run with an outdated configuration I suppose

Reply:
It's pretty annoying if you want to run clippy but `cargo make` does not let you run `clippy` because the check failed.

Reply:
well the commend wouldn't run, because `check` returns an exit code `1`, but I get your second point

Reply:
otherwise, (I am not as proficient in GitHub workflow magic), if the git repo is dirty after a task, would the GitHub action fail?

Reply:
Yes, it will run a `git diff`, prints the diff and error

Reply:
🤝

Reply:
Actually, we only do this for the `test` job. But you could add this snippet to the end of the `lint` job as well:

```yaml
      - name: Ensure empty git diff
        run: git --no-pager diff --exit-code --color
```